psng/psng.py

```diff
@@ -12,14 +12,14 @@
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 # GNU General Public License for more details.
 
-import hal                  # base hal class to react to hal signals
-import os                   # needed to get the paths and directorys
-import hal_glib             # needed to make our own hal pins
-import gtk                  # base for pygtk widgets and constants
+import hal  # base hal class to react to hal signals
+import os  # needed to get the paths and directorys
+import hal_glib  # needed to make our own hal pins
+import gtk  # base for pygtk widgets and constants
 import gtk.glade
-import sys                  # handle system calls
-import linuxcnc             # to get our own error sytsem
-import gobject              # needed to add the timer for periodic
+import sys  # handle system calls
+import linuxcnc  # to get our own error sytsem
+import gobject  # needed to add the timer for periodic
```

Comment:
can black-tool place the comments in the column?
(doest not really matter, but i want to test this review-tool :)

Reply:
Nope, that's one of the nice things about `black` - it's the python software foundation folks saying "This is how it should look - now you don't need to argue about what style to use - because the tool simply doesn't offer you a choice :smile:"

https://github.com/psf/black